client/ayon_resolve/api/pipeline.py

```diff
@@ -79,11 +83,28 @@ def install(self):
         register_creator_plugin_path(CREATE_PATH)
         register_inventory_action_path(INVENTORY_PATH)
 
+        # DaVinci Resolve version >= 20
+        # Set api resolve modules from undocumented injected cached app.
+        # https://forum.blackmagicdesign.com/viewtopic.php?f=21&t=113252
+        try:
+            bmdvf = self._app
+            bmdvr = self._app.GetResolve()
+            assert bmdvr and bmdvf
```

Comment:
Is `assert` the right approach here? Since they can be skipped in certain python run modes, etc. etc.
I know @iLLiCiTiT always likes to slap me in the face about this so can likely explain better.